app/app/users/page.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+import { useRouter } from "next/navigation";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Spinner from "react-bootstrap/Spinner";
+import Table from "react-bootstrap/Table";
+import AppBreadCrumb from "@/components/AppBreadCrumb";
+import { useUsersInfinite } from "@/utils/users";
+import { useToken } from "@/utils/auth";
+import AlignedLabel from "@/components/AlignedLabel";
+import { FaUserPlus, FaCopy } from "react-icons/fa6";
+
+export default function Users() {
+  const token = useToken();
+  const router = useRouter();
+  const { users, isLoading, error } = useUsersInfinite(token);
+
+  if (error) {
+    console.error(error);
+  }
+
+  return (
+    <>
+      <AppBreadCrumb />
+      <Card>
+        <Card.Header className="fs-5 fw-bold">Users</Card.Header>
+        <Card.Body>
+          <div className="mb-3">
+            {!isLoading && (
+              <>
+                <Button className="me-2">
+                  <AlignedLabel>
+                    <FaUserPlus className="me-2" />
+                    <span>Add user</span>
+                  </AlignedLabel>
+                </Button>
+                <Button>
+                  {/* todo: https://github.com/cityofaustin/atd-data-tech/issues/20121 */}
+                  <AlignedLabel>
+                    <FaCopy className="me-2" />
+                    <span>Copy user emails - todo</span>
+                  </AlignedLabel>
+                </Button>
+              </>
+            )}
+          </div>
+          {/* todo: standardize the way we show error messages and use error boundary */}
+          {Boolean(error) && (
+            <Alert variant="dange">Something went wrong</Alert>
```

Comment:
What would you think about putting the actual error in here? Maybe something like:


Maybe something like this?

```
{Boolean(error) && (
  <Alert variant="danger">Something went wrong: {error.message}</Alert>
)}
```

Reply:
(The error message in this case is just "Failed to fetch")